app/controllers/activities_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ActivitiesController < ApplicationController
+
+  before_action :set_project
+
+  def story
```

Comment:
Mmm, I see.
You can place this controller into a folder called `stories`.
It could go on `app/controllers/stories/activities_controller.rb`. And your class name goes like 
```Ruby
class Stories::ActivitiesController < ApplicationController
  def index
    ...
  end
end
```
And also, (if I'm not wrong) the `routes.rb` goes like:
```Ruby
resources :stories, only: [:index, :create, :update, :destroy, :show] do
  resources :activities, only: [:index]
  resources :notes
end
```